source/merge.js

```diff
@@ -15,17 +14,15 @@ import _curry2 from './internal/_curry2';
  * @param {Object} l
  * @param {Object} r
  * @return {Object}
- * @see R.mergeDeepRight, R.mergeWith, R.mergeWithKey
+ * @see R.mergeRight, R.mergeDeepRight, R.mergeWith, R.mergeWithKey
  * @example
  *
  *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
  *
- *      var resetToDefault = R.merge(R.__, {x: 0});
- *      resetToDefault({x: 5, y: 2}); //=> {x: 0, y: 2}
+ *      var withDefaults = R.merge({x: 0, y: 0});
+ *      withDefaults({y: 2}); //=> {x: 0, y: 2}
  * @symb R.merge({ x: 1, y: 2 }, { y: 5, z: 3 }) = { x: 1, y: 5, z: 3 }
  */
-var merge = _curry2(function merge(l, r) {
-  return _objectAssign({}, l, r);
-});
+var merge = mergeRight;
 export default merge;
```

Comment:
How about if we leave this one alone except to add "deprecated"/"see" tags to it?

It feels odd to be changing the implementation just in order to deprecate it.  I don't think the small amount of repeated code is an issue.

Reply:
Yeah I'm fine with that, updated 👍